suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -0,0 +1,144 @@
+# RBD: Persistent write back cache feature
```

Comment:
Add this suite in Pacific as well along with the pipeline metadata file.

Reply:
@sunilangadi2 , This is next step after this PR merge. And for this activity, need Amar's help.